src/screens/CreatePost/index.js

```diff
@@ -850,15 +844,13 @@ const CreatePost = () => {
             topics={listTopic}
             chatTopics={listTopicChat}
             onClose={() => sheetTopicRef.current.close()}
-            isAnonimity={typeUser}
             // saveOnClose={(v, chatTopic) => onSaveTopic(v, chatTopic)}
           />
           <SheetExpiredPost
             refExpired={sheetExpiredRef}
             data={postExpired}
             select={expiredSelect}
             onSelect={onSetExpiredSelect}
-            isAnonimity={typeUser}
           />
           <SheetGeographic
             geoRef={sheetGeoRef}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `isAnonimity` has been removed from multiple components. If these props were used in the child components, ensure they're no longer needed or are being provided through a different mechanism.
```diff
- isAnonimity={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->